packages/lit-dev-content/site/docs/getting-started.md

```diff
@@ -45,6 +45,48 @@ import {LitElement, html} from 'lit';
 
 {% endswitchable-sample %}
 
+## Use bundles
+
+Lit is also available as pre-built, single-file bundles. The bundles are
+standard JavaScript modules with no dependencies - any modern browser should be
+able to import and run the bundles directly.
```

Comment:
Should we add something like this so it's _super_ clear?
```suggestion
able to import and run the bundles from within a `<script type="module">`.
```

I know we're intentionally not choosing to support legacy script use cases and I think this is good, but I'm also not sure how common distributing an _ESM_ bundle is for the library bundle use case.

Reply:
sgtm